eval/src/vespa/eval/eval/dump_target.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <memory>
+#include <vector>
+#include <vespa/vespalib/stllike/string.h>
+
+namespace vespalib {
+namespace eval {
+
+struct TensorFunction;
+struct DumpTargetBackend;
+enum class Aggr;
+
+class DumpTarget
```

Comment:
Consider adding class comment

Reply:
Also consider making DumpTarget an interface to ease testing of TensorFunction::dump_tree().